src/e2e-test/java/io/cdap/cdap/ui/utils/Helper.java

```diff
@@ -87,4 +103,61 @@ public static String getNodeSelectorFromNodeIndentifier (NodeInfo node) {
       node.getNodeType() + "-" +
       node.getNodeId() + "\"]";
   }
+
+  public static void deployAndTestPipeline(String filename, String pipelineName) {
+    SeleniumDriver.openPage(Constants.BASE_STUDIO_URL + "pipelines");
+    WaitHelper.waitForPageToLoad();
+    ElementHelper.clickOnElement(locateElementById("resource-center-btn"));
+    ElementHelper.clickOnElement(locateElementById("create-pipeline-link"));
+    if(!SeleniumDriver.getDriver().getCurrentUrl().contains("studio")) {
```

Comment:
i couldn't find the exact rule but for java inside g3 but it looks like the preferred way is having a space after if statements https://google.github.io/styleguide/javaguide.html#s4.1.2-blocks-k-r-style

Reply:
yeah, changes done as suggested